recipes/rnablueprint/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+export BOOST_ROOT="${CONDA_PREFIX}"
+
+#libtoolize --copy --force
+#sh autogen.sh
+
+./configure \
+  --prefix=$PREFIX \
+  --disable-perl \
+  --disable-doxygen-doc \
+  --disable-program \
+  --disable-tests 
+
+make -j
```

Comment:
can you run `make test`?

Reply:
I included `make check -j` aafter `make -j` now